main.tf

```diff
@@ -52,7 +52,7 @@ resource "aws_iam_role_policy_attachment" "vpcflowlog-attach-localconfig-policy"
 resource "aws_flow_log" "cloudwatch" {
   count           = var.log_to_cloudwatch ? length(var.vpc_ids) : 0
   iam_role_arn    = aws_iam_role.this.arn
-  log_destination = aws_cloudwatch_log_group.this[count.index].arn
+  log_destination = aws_cloudwatch_log_group.this[0].arn
```

Comment:
What I read is that these are all logging to the same `aws_cloudwatch_log_group`, is that right?

Reply:
Correct. I didn't think it made sense to have a separate log group for each VPC as it'll be foldered by VPC ID anyway.